src/gui/PasswordGeneratorWidget.cpp

```diff
@@ -65,6 +79,7 @@ PasswordGeneratorWidget::~PasswordGeneratorWidget()
 
 void PasswordGeneratorWidget::loadSettings()
 {
+    // Password config
```

Comment:
@TheZ3ro we should remove the space at 87 to group all the password config then.